x/distribution/client/utils/utils.go

```diff
@@ -0,0 +1,48 @@
+package utils
```

Comment:
This should live in the `x/distribution/client/rest/` package

Reply:
There is a `utils` package like this in `x/params`, that's why I did it this way. What's the difference?

Reply:
(or should that one change too?)

Reply:
I actually second @alessio's comment here, I'd move the contents of this file to:
`rest/` (CommunityPoolSpendProposalReq) and 
`tx/` (CommunityPoolSpendProposalJSON and CommunityPoolSpendProposalJSON) 

This information is only used by those two packages, and thus doesn't really belong in it's own package - it's extra awkward here too because we `client/common` AND `client/utils`! 

In addition, I suggest these changes are made to remove `params/utils` as well (as you suggested @cwgoes) - it seems that params is an outlier in the pattern of defining `XXXRequest` structures outside of `...client/rest`  directory path.

Reply:
Sure respective types can be moved into `/rest` and `/tx` accordingly.

I wouldn't bother updating `x/params` here -- save that for another PR. However, `x/params` will still need some utility functionality (`ToParamChanges`).

Reply:
Ok, types moved into `/rest` and `/tx` appropriately.